src/controllers/shared.controllers/director.correspondence.address.controller.ts

```diff
@@ -110,7 +123,6 @@ export const postDirectorCorrespondenceAddress = async (req: Request, res: Respo
       return redirectToPath(path, req, res, lang);
     }
 
```

Comment:
I removed this second patchOfficerFiling() as no changes were made since line 95.